hugo/layouts/shortcodes/url.html

```diff
@@ -0,0 +1 @@
+{{ .Permalink }}
```

Comment:
what do we need this shortcode for? .Permalink is so simple you dont need to have a short code for it.. Also hugo has its own build in shotcodes for references and such. So I think this shortcode can be removed and just use {{ .Permalink }}in the template where it needed.